.nvmrc

```diff
@@ -0,0 +1 @@
+14
```

Comment:
?

Reply:
This indicates to `nvm use` that we will use node 14.x :-)